translations/strings.yaml

```diff
@@ -563,3 +563,11 @@ freeTrials:
     value: Sign up to start your 7-day free trial.
     comment: This string appears above the 'Sign up' button on mobile onboarding views
   freeTrialLabel: 7-day free trial
+
+general:
+  expandCard:
+    value: Expand card
```

Comment:
@flodolo Do you know if there are any conventions for this sort of "See more" / "See less" language in Firefox or Moz.org?

Reply:
Normally I see [expand](https://transvision.flod.org/?recherche=expand&repo=gecko_strings&sourcelocale=en-US&locale=en-US&search_type=strings_entities) and [collapse](https://transvision.flod.org/?recherche=collapse&repo=gecko_strings&sourcelocale=en-US&locale=en-US&search_type=strings_entities).